### PR TITLE
test: add cli integration test

### PR DESCRIPTION
**Pull request by mhevery**

### CODE REVIEW

File: integration/ng-cli-create.sh [hunk 1]
```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+
+set -e -o pipefail
+
+if [ $# -eq 0 ]
+  then
+    echo "Angular cli integration create project"
+    echo
+    echo "./ng-create.sh [project-name]"
```
Comment (tinayuangao) [inline integration/ng-cli-create.sh]: Is it "ng-cli-create.sh"?

File: integration/run_tests.sh [hunk 2]
```diff
@@ -14,6 +14,15 @@ rm_cache
 mkdir $cache
 trap rm_cache EXIT
 
+if [ ! -d ".ng-cli" ]; then
```
Comment (tinayuangao) [inline integration/run_tests.sh]: Add comments about why we need this part?

File: integration/ng-cli-create.sh [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  NG=$INTEGRATION_DIR/.ng-cli/node_modules/.bin/ng
+  (
+    cd $INTEGRATION_DIR
+    rm -rf $PROJECT
+    $NG set --global packageManager=yarn
+    $NG new $PROJECT --skip-install
+  )
+  (
+    cd $PROJECT_DIR
+    yarn add \
```
Comment (tinayuangao) [inline integration/ng-cli-create.sh]: Add comments about why adding these dependencies?

File: integration/ng-cli-create.sh (hunk 1 above)
Comment (mhevery) [inline integration/ng-cli-create.sh]: fixed

File: integration/ng-cli-create.sh (hunk 3 above)
Comment (mhevery) [inline integration/ng-cli-create.sh]: comment added

File: integration/run_tests.sh (hunk 2 above)
Comment (mhevery) [inline integration/run_tests.sh]: added

**Review by tinayuangao — APPROVED** [review]
LGTM

**Pull request closed without merging**